frontend/src/pages/admin/users.tsx

```diff
@@ -8,12 +8,30 @@ import Pagination from '~/components/admin/Pagination'
 import { adminProtected } from '~/utils/admin-protected'
 
 const Users: NextPage = (): JSX.Element => {
+  const [loading, setLoading] = useState(false)
+
+  const [pageNumber, setPageNumber] = useState(0)
+
+  const userPerPage = 5
+  const pagesVisited = pageNumber * userPerPage
+
+  const displayUsers = users?.slice(pagesVisited, pagesVisited + userPerPage)
+
+  const pageCount = Math.ceil(users?.length / userPerPage)
+
+  const changePage = ({ selected }: { selected: number }): void => setPageNumber(selected)
```

Comment:
```suggestion
  const changePage = (selected: number): void => setPageNumber(selected)
```

please check if this will work